CityOfDetroit/bloom
# 1442/listing finder q3

# Pull Request Template

## Issue Overview

This PR addresses #1442 


## Description

This PR builds out q3 of the listings finder. Since q3 introduced a new pattern, it warranted a refactor of the finder to utilize separate custom components rather than trying to contain all the cases within finder.tsx. 

## How Can This Be Tested/Reviewed?

This can be tested by clicking through the form and ensuring that the form saves and applies the rental cost information entered whether you click previous on the disclaimer page or finish and see it reflected in the listings.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
